esx_service/utils/auth.py

```diff
@@ -28,6 +28,7 @@
 CMD_REMOVE = 'remove'
 CMD_ATTACH = 'attach'
 CMD_DETACH = 'detach'
+CMD_LIST = 'list'
```

Comment:
In my unit test code, I need to test 'list' command. It seems that your change assume we only put commands which need authorize in auth.py. I will remove 'list' command from this file and update the comment.